content/case_histories/bookpurnong/interpretation.rst

```diff
@@ -2,3 +2,77 @@
+
+(4) Plate modeling. Some geologic targets can be characterized as conductive thin plates that give rise complicated EM responses due to the geometry and mutual coupling (Link to 3-loop). Plate modeling method attempts to find the geometry and conductivity of a few conducting plate in a uniform hackground that is responsible for most of the anomalous data. It has the advantages of being able to handle the 3D coupling effect efficiently, but may have trouble dealing with too many plates in more complex situations. Link to plate modeling page.
+
+(5) 1D layered earth model inversion. This approach assume the earth's codnuctivity only varies as a function of depth. At each measurement location, the inversion find a layered model that explains the entire decay curve in time or the entire spectrum in frequency-domain. Many layered models at multiple locations then can be stitched together to form a pseudo-3D volume for visualization. Advanced techniques also consider the correlation between adjacent locations by imposing lateral constraints, etc. Link to 1D inversion page.
+
+(6) Full 3D inversion. The previous interpreting methods all assume the earth has a particular structure so simplified calculations can be used. Any violation of those assumptions would result in failures. A 3D inversion discretizes the entire earth to many discrete cells, each of which has a constant conductivity. Then the Maxwell's equations are solved on the mesh. The obtained images of the subsurface are in 3D voxel format. 3D inversions provides the best resolution and works for any complicated models, but it is more computational expensive. Link to 3D inversion page.
+
+The geology at Bookpurnong is mostly flat strata of floodplain. Therefore, an 1D inversion would be the most cost-effective solution for the data interpretation. In the following, we will go through the 1D inversion of the three data sets. 
+
+
+1D inversion of RESOLVE data
+----------------------------
+
+For RESOLVE data, the earth is discretized into 21 layers over the basement; the layers' thickness logarithmically increases from 0.5 to 43 m. Two different values, 1 and 0.1 S/m, are used as the conductivity of the uniform background model for two independent inversion trials. 
+
+.. figure:: ./images/booky-resolvefit.jpg
```

Comment:
I like the figure but need a legend. What are the red vs blue? continue vs dashed lines? (pred. vs obs ; In-Phase / quadrature? ) what represent the two axis and what are the units?